Description: Fix FTBFS on powerpc, ppc64 and ppc64el
 Chrony fails to build from source on these architectures when enabling syscall
 filtering feature due to missing terminal I/O interfaces header (termios.h)
 inclusion.
Author: Miroslav Lichvar <mlichvar@redhat.com>
Bug: https://listengine.tuxfamily.org/chrony.tuxfamily.org/chrony-dev/2016/03/msg00009.html
Applied-Upstream: https://git.tuxfamily.org/chrony/chrony.git/commit/?id=5645e57ce001daa3d63a7678cca306b8edfa005c
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sys_linux.c
+++ b/sys_linux.c
@@ -50,6 +50,7 @@
 #ifdef FEAT_SCFILTER
 #include <sys/prctl.h>
 #include <seccomp.h>
+#include <termios.h>
 #ifdef FEAT_PHC
 #include <linux/ptp_clock.h>
 #endif
